src/helpers/transactions.js

```diff
@@ -34,7 +34,7 @@ const calculateTimestampOfThisMonth = () => {
 
 const calculateTimestampOfThisYear = () => {
   var d = new Date();
-  d.setFullYear(d.getFullYear, 1, 1);
+  d.setFullYear(d.getFullYear(), 0, 0);
```

Comment:
The month is 0-11 so this fix is right but the last parameter should be 1 (goes from 1-31)

“Javascript” 😅

Reply:
🙏